web-api/src/storage/memory.rs

```diff
@@ -17,21 +17,27 @@
 //   "U") to allow for faster map access without hashing (ie each u32 is already its own hash)
```

Comment:
This is not true, an integer is hashed anyway using the internal state of the hasher and some constant. The only difference is in the number of bytes.

Having this wrapper limits the usability of this storage, making it a mind specific storage and not a general memory storage that could be used for other tests.

Reply:
> This is not true, an integer is hashed anyway using the internal state of the hasher and some constant. The only difference is in the number of bytes.

i forgot to update the comment, this refers to the identity hasher (which just passed through the `u32`) before `fnv` was added to replace it. i'll update the comments after the code cleanup.